core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -240,6 +244,34 @@ protected function renderTwigSurrogateIfExists(): string|null
         return $partialTemplate->parse();
     }
 
+    /**
+     * @see \Contao\CoreBundle\Twig\ResponseContext\AddNode
+     * @see \Contao\CoreBundle\Twig\ResponseContext\AddTokenParser
+     *
+     * @internal
+     */
+    public function addDocumentContent(string|null $identifier, string $content, DocumentLocation $location): void
+    {
+        // TODO: This should make use of the response context in the future.
```

Comment:
Why can it not make use of it now?

Reply:
Because this is currently not implemented yet in the response context. But @Toflar or myself will probably do this in the foreseeable future. :slightly_smiling_face:

Reply:
I'll probably give it a shot even for 5.0 - depending on the permissions stuff.